recipes/nb_conda_store_kernels/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "nb_conda_store_kernels" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8744e734a64154faf8e178bb1928e012d1fd940c7305f0751d0d0a2c3618367d
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - conda-store >=0.4.4
+    - jupyter_client
+    - traitlets
```

Comment:
Very minor nitpick @costrouc but it helps to parse the recipes visually and update them if the requirements are liste in alphabetical order :)

Reply:
Great in the future I will alphabetically sort them. Thanks for merging!